packaging/windows/OpenRA.nsi

```diff
@@ -170,6 +170,24 @@ Section "-DotNet" DotNet
 	done:
 SectionEnd
 
+Section "-MSVC100" MSVC100
+	ClearErrors
+	ReadRegDWORD $0 HKLM "Software\Microsoft\DevDiv\vc\Servicing\10.0\RuntimeMinimum" "Install"

```

Comment:
For me the "RuntimeMinimum" key is in "11.0", so this would give me a false positive.
Also this looks like it would give false positives in general, if the user has VC++ newer than 2010, but not 2010 (just guessing here).

I would suggest looking at #8360, because it checks for the exact same thing - whether VC++ 2010 or later is installed.